test/Manifest.toml

```diff
@@ -664,11 +684,17 @@ git-tree-sha1 = "cb7d8b805413025a5bc866fc036b426223ffc059"
 uuid = "63c18a36-062a-441e-b654-da1e3ab1ce7c"
 version = "0.7.2"
 
+[[deps.Kinematic1D]]
+deps = ["ClimaCore", "ClimaCorePlots", "CloudMicrophysics", "DiffEqBase", "Logging", "NCDatasets", "OrdinaryDiffEq", "Plots", "ProgressLogging", "TerminalLoggers", "Thermodynamics", "UnPack"]
+git-tree-sha1 = "e8a340e04e51a31367239095f97b6e7a697f7a8f"
+uuid = "a8a3bb05-6cc5-4342-abf6-5cc636bd2b35"
+version = "0.1.0"
```

Comment:
The `path` entry is not here, so this means that you `julia --project=test] add Kinematic1D`, which adds that specific version (`e8a340e04e51a31367239095f97b6e7a697f7a8f`). I think what you want here is to `julia --project=test] dev .` from `Kinematic1D.jl/`.

Reply:
Looks like with `dev .` I'm running into an error: `ERROR: LoadError: can not merge projects` (https://github.com/CliMA/Kinematic1D.jl/runs/7677415528?check_suite_focus=true)

I saw you are familiar with it: https://github.com/CliMA/TurbulenceConvection.jl/pull/864

For example, in CloudMicrophysics.jl I just don't check in the `test/Manifest.toml` file. Could I do it here as well?